src/main/java/edu/cursor/service/PlaceServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package edu.cursor.service;
+
+import edu.cursor.entities.Place;
+import edu.cursor.exceptions.DataNotFoundException;
+import edu.cursor.repository.PlaceRepository;
+import lombok.AllArgsConstructor;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class PlaceServiceImpl implements PlaceService {
+    private final PlaceRepository placeRepository;
+
+    @Override
+    public Place addPlace(Place place) {
+        return placeRepository.save(place);
+    }
+
+    @Override
+    public void deletePlaceById(Long id) {
+        placeRepository.deleteById(id);
+    }
+
+    @Override
+    public Place findPlaceById(Long id) {
+        return placeRepository.findById(id).orElseThrow(() -> new DataNotFoundException("The place doesn't found"));
```

Comment:
Create a separate interface such as "Messages" and move all messages to it. Try to avoid plain text in code 😉